CONTRIBUTING.md

```diff
@@ -543,10 +533,47 @@ Refer datalad/config.py for information on how to add these environment variable
 - *DATALAD_ALLOW_FAIL*:
   Instructs `@never_fail` decorator to allow to fail, e.g. to ease debugging.
 
-# Changelog section
+# Release(s) workflow
+
+## Branches
+
+- `master`: changes toward the next `MAJOR.MINOR.0` release (e.g., `0.13` as of time of writing).
+  It is also used for release candidates (with `rcX` suffix)
+- `maint`: bug fixes and minor enhancements for the latest released `MAJOR.MINOR.PATCH`
```

Comment:
I'd prefer to drop the "minor enhancements" part.  Of course, they can be argued on a case by case base, but I don't see any reason to suggest they should go to maint.